res/values/strings.xml

```diff
@@ -461,9 +461,11 @@
 
 
     <!-- welcome and login -->
+    <string name="welcome_chat_over_email">Chat over E-Mail.</string>
+    <string name="scan_invitation_code">Scan Invitation Code</string>
     <string name="welcome_intro1_message">The messenger with the broadest audience in the world. Free and independent.</string>
     <string name="login_title">Log In</string>
-    <string name="login_header">Log In to Your Server</string>
+    <string name="login_header">Log into your E-Mail Account</string>
```

Comment:
or maybe better `Log Into Your E-Mail Account`?

(the button all-uppercase comes from the system, however, the line will be also used as title for the subsequent screen)